src/main/java/pl/trainingapp/repositories/ExerciseRepository.java

```diff
@@ -0,0 +1,38 @@
+package pl.trainingapp.repositories;
+
+import org.springframework.stereotype.Repository;
+import pl.trainingapp.entities.Exercise;
+import pl.trainingapp.exceptions.EntityNotFoundException;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class ExerciseRepository {
+    
+    public void addExercise(String name) {
+        int maxId = ExerciseDatabase.exercises.stream().mapToInt(Exercise::getId).max().orElse(1);
+
+        ExerciseDatabase.exercises.add(new Exercise(maxId, name));
```

Comment:
Logic for creating Entity should be in service and repository should be only responsible for saving it in database. But let's leave it here for now, later on we will move it to it's designated place